subprojects/plugins/src/main/java/org/gradle/api/plugins/jvm/TestFixturesDependencyModifiers.java

```diff
@@ -55,25 +55,26 @@ public interface TestFixturesDependencyModifiers {
      * @see #modify(ModuleDependency)
```

Comment:
❓ I think this needs to link to modifyImplementation now? All of the modifiers have this identical javadoc.